docs/docs-content/release-notes/known-issues.md

```diff
@@ -16,6 +16,7 @@ The following table lists all known issues that are currently active and affecti
 | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | -------------- | --------------------- |
+| When provisioning an Edge cluster, it's possible that some Operating System (OS) user credentials will be lost once the cluster is active. This is because the cloud-init stages from different sources merge during the deployment process and sometimes the same stages without distinct names overwrite each other.                                                                                                                                                                       | Give each of your cloud-init stages in the OS pack and in the Edge installer **user-data** file a unique name. For more information about cloud-init stages and examples of cloud-init stages with names, refer to [Cloud-init Stages](../clusters/edge/edge-configuration/cloud-init#erase-partitions.md).                                                                                                                                                                                                                                                                                                     | July 17 2024   | Edge                  |
```

Comment:
```suggestion
| When provisioning an Edge cluster, it's possible that some Operating System (OS) user credentials will be lost once the cluster is active. This is because the cloud-init stages from different sources merge during the deployment process, and sometimes, stages with the same names overwrite each other.                                                                                                                                                                       | Give each of your cloud-init stages in the OS pack and in the Edge installer **user-data** file a unique name. For more information about cloud-init stages and examples of cloud-init stages with names, refer to [Cloud-init Stages](../clusters/edge/edge-configuration/cloud-init#erase-partitions.md).                                                                                                                                                                                                                                                                                                     | July 17 2024   | Edge                  |
```
Maybe? :)
Also, I think the cloud-init stages link is broken